v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/schema/ISchemaMapper.java

```diff
@@ -25,6 +25,14 @@ public interface ISchemaMapper extends Serializable {
   /** Retrieves the Spanner dialect. */
   Dialect getDialect();
 
+  /**
+   * Return the source tables configured for migration.
+   *
+   * @param namespace is currently not operational.
+   * @return
+   */
+  List<String> getSourceTablesToMigrate(String namespace);
```

Comment:
should this just be `getSourceTables` keeping it consistent with the namign convention of the remaining mthods?